PWGGA/GammaConv/AliAnalysisTaskGammaCalo.cxx

```diff
@@ -6520,11 +6520,16 @@ void AliAnalysisTaskGammaCalo::CalculateBackground(){
             }
           }
           if(JetNearEMCal){
-            Double_t zbin = fBGHandler[fiCut]->GetZBinIndex(2);
```

Comment:
Do you really want to get rid of this?

Reply:
Yes because I now I want to mix with the actual z bin instead of trowing all background candidates in the same z bin. And the variable definition is also not a problem because it is defined at the top of the CalculateBackground function

Reply:
Ok, just wanted to make sure this is not an unwanted removal